lib/features/thread/presentation/mixin/base_email_item_tile.dart

```diff
@@ -25,6 +25,39 @@ mixin BaseEmailItemTile {
   final responsiveUtils = Get.find<ResponsiveUtils>();
   final imagePaths = Get.find<ImagePaths>();
 
+  Widget buildIconAvatarSwipe({
+    double? iconSize,
+    TextStyle? textStyle
+  }) {
+    return Container(
+      width: iconSize ?? 56,
+      height: iconSize ?? 56,
+      alignment: Alignment.center,
+      decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular((iconSize ?? 56) * 0.5),
+          border: Border.all(color: Colors.transparent),
+          color: AppColor.colorSpamReportBox
+      ),
```

Comment:
```suggestion
      decoration: BoxDecoration(     
        borderRadius: BorderRadius.circular((iconSize ?? 56) * 0.5),
        border: Border.all(color: Colors.transparent),
        color: AppColor.colorSpamReportBox
      ),
```

Reply:
`borderRadius: BorderRadius.circular((iconSize ?? 56) * 0.5),`
Why did you use it?

Reply:
done